source/opt/reflect.h

```diff
@@ -26,12 +26,14 @@ namespace ir {
 
 inline bool IsDebug1Inst(SpvOp opcode) {
   return (opcode >= SpvOpSourceContinued && opcode <= SpvOpSourceExtension) ||
-         opcode == SpvOpString || opcode == SpvOpLine ||
-         opcode == SpvOpNoLine || opcode == SpvOpModuleProcessed;
+         opcode == SpvOpString || opcode == SpvOpLine || opcode == SpvOpNoLine;
```

Comment:
We cannot have `OpLine` and `OpNoLine` until the type-consant-variable section (9). So So I assume debug1 instructions should not contain them?